### PR TITLE
Some small refactors on the Game class

### PR DESCRIPTION
**Pull request by Fito**
- Converts tabs into spaces in Game.cs (keeps for spaces)
- Makes scene variable private
- sets scene to "Login" on Awake

### CODE REVIEW

**Comment by Fito** [comment]
@jensvdh Ready for review

File: Assets/Game.cs [hunk 1]
```diff
@@ -4,148 +4,149 @@
...
+    void Awake ()
+    {
+        scene = "Login";
+        DontDestroyOnLoad (gameObject);
+
+        SpeciesTable.Initialize ();
+        texture = Functions.CreateTexture2D (Color.black);
+    }
+
+  // Use this for initialization
```
Comment (jensvdh) [inline Assets/Game.cs]: @Fito 

Can you re-indent all the comments. IDK what happened to those.

They usually have the same indentation as code on the same level.

For comments there's usually one space after "//", regular indentation before it.

LMK when that's done and we'll merge the PR into dev.

Comment (Fito) [inline Assets/Game.cs]: @jensvdh
Done

**Comment by jensvdh** [comment]
:+1: 

Merged.

**Pull request merged**